### ISSUE TITLE
Update user profile doesn't seem to do anything

### ISSUE THREAD

**Issue by Nithyavasudevan** [okfn/ckanext-glasgow#167]
1. Went to Organisations page and selected an organisation and clicked on manage.
2. Selected MEmbers and selected a member.
3. Clicked on Manage in the member page. It fetches the username with domain so any changes do not get submitted.
4. However, when i remove the domain part and make changes to user profile and submit, it seems to have made changes locally and the changes are not getting sent to platform. 
5. Also, not sure if the change password section in this page does anythin

![image](https://cloud.githubusercontent.com/assets/7315913/4189640/1c4373e6-377e-11e4-95ad-fd5065627bd8.png)

**Comment by adamamyl** [comment]
[Thoughts]: drop the username field, or make it non-editable (like quite a lot of sites do).

Changing password is going to be tricky: for the two types of user (CTPEC & CKAN-local).

I think the best action might be (assuming [WAAD supports changing passwords](https://www.yammer.com/futurecitiesdemonstrator/#/Threads/show?threadId=437548704)) is to have two options: 'Citizen' (for CKAN) and 'Public Servant' (for CTPEC) (or some other label).

**Comment by Nithyavasudevan** [comment]
let's discuss this on the call

**Comment by adamamyl** [comment]
Change password uses a different call.

**Comment by mcginns** [comment]
This is blocking the closure of user story #54 (update of user account details). Can we re-categorise as a P1 for investigation pls?

User profile information update fails due to a validation problem on the Username field:

![image](https://cloud.githubusercontent.com/assets/7511445/4303313/e0b058e2-3e64-11e4-9973-7748a13bd6d2.png)

**Comment by mcginns** [comment]
I get a platform error referencing a null entry for userid when I try to update description for user David Cameron as super user on https://ckanfrontend.cloudapp.net

![image](https://cloud.githubusercontent.com/assets/7511445/4402478/001d54ee-4498-11e4-9f1b-a7005a027d50.png)

**Comment by joetsoi** [comment]
@mcginns I've updated the code on integration again and the requests now get sent but when you check it, it will fail with a validation error, are the microsoft guys still about for us to see what happened on their end?

**Comment by mcginns** [comment]
Yeah update request seems to go ok but the change isn't harvested back to the user profile. 
Jakub finished on the project on Friday but I'm sure he'll take a look at the logs for you. 

Drop something onto Yammer and ask if he could take a quick look.
